tests/scripts/run-throughput-perf.py

```diff
@@ -290,6 +295,11 @@ def runIterations(dll_name, dll_path, iterations, crossgen_path, jit_path, assem
     if opt_level == 'min_opt':
         my_env['COMPlus_JITMinOpts'] = '1'
 
+    if jit_name == 'legacy':
+        my_env['COMPLUS_AltJit'] = '*'
```

Comment:
The way we did this in the past was to give crossgen a different JitPath for the legacy backend run. You may still need to pass AltJit/AltJitNgen, etc. but we had working runs when passing legacyjit.dll to crossgen as the jit. Have you tested this to make sure the timing data looks about right?